packages/core/lib/types/queueOptionsTypes.ts

```diff
@@ -35,15 +36,13 @@ export type DeletionConfig = {
 
 type NewQueueOptions<CreationConfigType extends CommonCreationConfigType> = {
   locatorConfig?: never
-  deletionConfig?: DeletionConfig
   creationConfig: CreationConfigType
-} & CommonQueueOptions
+}
 
 type ExistingQueueOptions<QueueLocatorType extends object> = {
   locatorConfig: QueueLocatorType
-  deletionConfig?: DeletionConfig
   creationConfig?: never
-} & CommonQueueOptions
+}
```

Comment:
`CommonQueueOptions` is already being used on `QueueOptions`, so avoiding repetition